<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RdfCalendarEventEditWidget</class>
 <widget class="QWidget" name="RdfCalendarEventEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>462</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLineEdit" name="summary"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Summary</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="location"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="endDate"/>
     </item>
     <item>
      <widget class="QTimeEdit" name="endTime"/>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tadtab">
      <attribute name="title">
       <string>Time and Date</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="KDatePicker" name="startDate"/>
       </item>
       <item row="0" column="0">
        <widget class="QTimeEdit" name="startTime"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tztab">
      <attribute name="title">
       <string>Timezone</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="KTimeZoneWidget" name="tz">
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTimeZoneWidget</class>
   <extends>QTreeWidget</extends>
   <header>ktimezonewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDatePicker</class>
   <extends>QFrame</extends>
   <header>kdatepicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>startDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>endDate</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>endTime</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
